src/index.html

```diff
@@ -22,6 +22,349 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="header">
+        <a
+          class="logo"
+          href="index.html"
+        >
+          <img
+            src="/src/images/logo.png"
```

Comment:
The path to the image should not start with '/src' when referencing from HTML. It should be a relative path like 'images/logo.png'.